src/DotLiquid/Tokenizer.cs

```diff
@@ -169,7 +184,7 @@ internal static IEnumerator<string> GetVariableEnumerator(string source)
         /// </summary>
         /// <param name="markupEnumerator">The string enumerator</param>
         /// <param name="markupLength">The number of characters to read</param>
-        private static string ReadChars(CharEnumerator markupEnumerator, int markupLength)
+        internal static string ReadChars(CharEnumerator markupEnumerator, int markupLength)
```

Comment:
I'm always wary when a private method needs to be made accessible. This is usually a smell that we're tying classes together for convenience that may not have a real connection if designed from scratch. Same for `ReadToEndofTag`.
Also, why is this internal and the other is public (on an internal class)?